src/messages.rs

```diff
@@ -267,6 +263,9 @@ impl RoutingMessage {
 #[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug, RustcEncodable, RustcDecodable)]
 pub struct SignedMessage {
     encoded_body : Vec<u8>,
+    claimant     : Address,
```

Comment:
if it is ManagedNode, then yes it is redundant;
when it is client, it is not undefined, because Address is an enum {Node(nametype), Client(PublicKey)}

the only comment I have is that Address seems to be a misnomer, I'd prefer "Identity", but (re)naming isnt a priority just now